src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -1106,6 +1106,20 @@ protected void ExecuteIndexing()
                                         var originalName = Name.Replace(Constants.Documents.Indexing.SideBySideIndexNamePrefix, string.Empty, StringComparison.InvariantCultureIgnoreCase);
                                         _isReplacing = true;
 
+                                        if (batchCompleted)
+                                        {
+                                            // this side-by-side index will be replaced in a second, notify about indexing success
+                                            // so we know that indexing batch is no longer in progress
+                                            DocumentDatabase.Changes.RaiseNotifications(new IndexChange
+                                            {
+                                                Name = Name,
+                                                Type = IndexChangeTypes.BatchCompleted
+                                            });
+                                            
+                                            var batchCompletedAction = DocumentDatabase.IndexStore.IndexBatchCompleted;
+                                            batchCompletedAction?.Invoke((Name, didWork));
```

Comment:
Let's extract this piece of code to a method. We have exactly the same a few lines code below.

Reply:
done